spec/config.md

```diff
@@ -32,6 +32,28 @@ chains:
       - prefix: wasm
       # this determines the gas payments we make (and defines the fee token)
       - gas_price: 0.1umayo
+      # this defines the gas limits we are going to use
```

Comment:
This no longer exists in the registry.yaml.
We should update this to point to oysternet